src/index.html

```diff
@@ -13,5 +13,237 @@
+        <legend>Registration</legend>
+
+        <label class="form__field">
+          <label
+            for="your-email">
+            E-mail:
+          </label>
+          <input
+            type="email"
+            name="email"
+            id="your-email"
+            placeholder="email@example.com"
+            pattern="@"
+            required
+          >
+        </label>
+
+        <label class="form__field">
+          <label
+            for="your-pasaword">
+            Password:
+          </label>
+          <input
+            type="password"
+            name="password"
+            id="your-pasaword"
+            minlength="5"
+            maxlength="10"
+            pattern="\w+"
+            required
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="group-vertical-distance">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="form__field">
+          Do you love cats?
+          <input
+            type="radio"
+            name="radio"
+          >
+          Yes
+          <input
+            type="radio"
+            name="radio"
+          >
+          No
+        </label>
+
+        <label class="form__field">
+          <label 
+            for="favcolor">
+            What is your favorite color?
+          </label>
+          <input type="color" name="color" id="favcolor" required>
+        </label>
+
+        <label class="form__field">
+          <label 
+            for="badtime">
+            What time do you go to bad?
+          </label>
+          <input
+            type="time"
+            name="time"
+            id="badtime"
+            step="2"
+            required
+          >
+        </label>
+
+        <label class="form__field">
+          <label
+            for="favorite-car">
+            What are your favorite brand of cars?
+          </label>
+          <select
+            name="color"
+            id="favorite-car"
+            required
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form_field">
+          <label
+            for="rate">
+            How do you rate our work?
+          </label>
+          <input 
+            type="range"
+            name="range"
+            id="rate"
+            min="0"
+            max="70"
+            required
+          >
+        </label>
+      
+      </fieldset>
+
+      <fieldset class="group-vertical-distance">
+        <legend>Additional info</legend>
+
+        <label class="form__field">
+          <label 
+            for="your-comments">
+            Comments:
+          </label>
+          <textarea
+            name="textarea"
+            id="your-comments"
+            cols="17"
+            rows="2"
+            required
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea
            name="textarea"
            id="your-comments"
            cols="17"
            rows="2"
            required
          ></textarea>
```
Саме з textarea треба так робити, бо якшо написати так як ти, друк починається десь з середини